<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetWindow</class>
 <widget class="QWidget" name="SetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBegin">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>710</y>
     <width>81</width>
     <height>80</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/img_icon/start_icon.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>请选择你的角色</string>
   </property>
   <widget class="QRadioButton" name="radioButton11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IKUN</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>郭神</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>路人甲</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>绿头鸭</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>550</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>请选择你的角色</string>
   </property>
   <widget class="QRadioButton" name="radioButton21">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>IKUN</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton22">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>郭神</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton23">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>路人甲</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton24">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>绿头鸭</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="BackGround">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/img_BG/prepareBG_800.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_intro1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>260</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;IKUN 只因你太美&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;敏捷(2) 弹跳(5) 力量(2)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_intro2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>390</y>
     <width>260</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;IKUN 只因你太美&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;敏捷(2) 弹跳(5) 力量(2)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_role1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/img_role/ikunW.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_role2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>100</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/img_role/ikunW.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>550</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>请选择场地</string>
   </property>
   <widget class="QRadioButton" name="radioButton31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>115</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>羽毛球馆</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>115</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>月球基地</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton33">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>115</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>太极道场</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_playground_text">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;一个普通的羽毛球馆&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;重力(3) 空气阻力(3)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_playground_pic">
   <property name="geometry">
    <rect>
     <x>508</x>
     <y>132</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/img_BG/gymBG_260.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>BackGround</zorder>
  <zorder>pushButtonBack</zorder>
  <zorder>pushButtonBegin</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_intro2</zorder>
  <zorder>label_intro1</zorder>
  <zorder>label_role1</zorder>
  <zorder>label_role2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>label_playground_text</zorder>
  <zorder>label_playground_pic</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
